docs/kura-wires/single-port-wire-components/db-store-and-filter.md

```diff
@@ -1,23 +1,36 @@
 
-- **sql.view**: SQL to be executed to build a view.
-- **cache.expiration.interval**: cache validity in seconds. When the cache expires, a new read in the database will be performed.
-- **DbService Target Filter** : the database instance to be used.
-- **emit.empty.result** : defines if the envelope should be emitted even if the query return an empty result.
+    The reason of the deprecation is the fact that Db Store and Db Filter only support databases that provide a JDBC interface. Moreover, the old Db Store interacts with the database using a set of hardcoded SQL queries.
+    This fact makes it difficult to use it with different databases since the syntax of the queries is usually database-specific.
+    
+    The new components use the `org.eclipse.kura.wire.store.provider.WireRecordStoreProvider` and `org.eclipse.kura.wire.store.provider.QueryableWireRecordStoreProvider` APIs introduced in 5.3.0 allowing to use them with generic data store implementations.
 
-The following procedure will create a wire graph that collects data from a simulated OPC-UA Server, stores it in a table in the database, using the DB Store component, and publishes it in the cloud platform. Moreover, the DB Filter is used to read from the database and write data to the OPC-UA Server based on the values read.
+    Please note that Wire Record Query component is not portable by nature, since it allows to execute an arbitrary user defined query. The new APIs allows to use it with non-JDBC data stores.
 
+The Wire Record Store component can be configured as follows:
+
+- **Record Collection Name**: The name of the record collection that should be used. The implementation of the collection depends on the Wire Record Store implementation, if it is a SQL database, the collection will likely be a table.
+- **Maximum Record Collection Size**: The maximum number of records that is possible to store in the collection.
+- **Cleanup Records Keep**: The number of records in the collection to keep while performing a cleanup operation (if set to 0 all the records will be deleted). The cleanup operation is performed when a new record is inserted and the current size of the record collection is greater than the configured **Maximum Record Collection Size**.
+- **WireRecordStoreProvider Target Filter** : Specifies, as an OSGi target filter, the pid of the of the Wire Record Store instance to be used.
+
+The Wire Record Query component can be configured as follows:
+
+- **Query**: Query to be executed. The query syntax depends on the Queryable Wire Record Store implementation.
+- **Cache Expiration Interval (Seconds)**: This component is capable of maintaining a cache of the records produced by the last query execution and emitting its contents on the Wire until it expires. This value specifies the cache validity interval in seconds. When cache expires, it will cause a new read in the database. A database read will be performed for every trigger received if the value is set to 0.
```

Comment:
When cache expires, it will cause a new read in the **collection**. A **collection** read will be performed for every trigger received if the value is set to 0.